doc.go

```diff
@@ -187,6 +187,14 @@ last.
 	$ ron target ron:prep
 	$ ron target prep
 
+ron will attempt to find any additional files in the following order:
+
+	1. A -yaml flag filepath config file
+	2. A local ./ron.yaml or in a parent directory
+	3. Any files in the folder ./.ron/*.yaml
```

Comment:
I don't have any ideas on a use case for another type of yaml config. If a special case does come up for ~/.ron a subfolder seems fine with me